packages/builders/scripts/inter-protocol/updatePriceFeeds.js

```diff
@@ -34,6 +34,14 @@ const configurations = {
     ],
     inBrandNames: ['ATOM', 'stTIA', 'stkATOM'],
   },
+  EMERYNET: {
+    oracleAddresses: [
+      'agoric1ldmtatp24qlllgxmrsjzcpe20fvlkp448zcuce', // GOV1
+      'agoric140dmkrz2e42ergjj7gyvejhzmjzurvqeq82ang', // GOV2
+      'agoric1f0h5zgxyg3euxsqzs0506uj4cmu56y30pqx46s', // GOV4
```

Comment:
@mhofman only gov1 and gov2 serve as oracle addresses on emerynet. What would this entail? Should we remove this?